pyspeckit/spectrum/models/ammonia.py

```diff
@@ -176,44 +178,50 @@ def ammonia(xarr, tkin=20, tex=None, ntot=14, width=1, xoff_v=0.0,
         comment each step carefully.
         """
         Zpara = (2*Jpara+1)*np.exp(-h*(Brot*Jpara*(Jpara+1)+
-                                       (Crot-Brot)*Jpara**2)/(kb*tkin))
+                                       (Crot-Brot)*Jpara**2)/(kb*trot))
```

Comment:
Well... technically we're not assuming any levels will be specified here; this could be T_rot between (5,5) and (13,13), for all we know.  But, yes, we are assuming a single rotational temperature for all lines.